src/cmake/FindAPR.cmake

```diff
@@ -33,7 +33,14 @@ find_program(APR_CONFIG_EXECUTABLE
 mark_as_advanced(APR_CONFIG_EXECUTABLE)
 if(EXISTS ${APR_CONFIG_EXECUTABLE})
     _apr_invoke(APR_INCLUDE_DIR  --includedir)
-    _apr_invoke(APR_LIBRARIES  --link-ld)
+    if (APR_STATIC OR NOT BUILD_SHARED_LIBS)
+      _apr_invoke(_apr_link_args  --link-ld)
+      string(REGEX MATCH "-L([^ ]*)" _apr_L_flag ${_apr_link_args})
```

Comment:
Same like above regarding the regular expression.

Reply:
The regex is designed to work with the output generated by the 'apr-1-config' installed by APR install.

If** the -L and the path are separated, the logic would need to find the argument after '-L'.